Makefile

```diff
@@ -161,6 +161,18 @@ test-full:
 	$(info log-file: test-$(TEST_SUFFIX).log)
 	PASSES="fmt build release unit integration functional e2e grpcproxy" ./test.sh 2<&1 | tee test-$(TEST_SUFFIX).log
 
+.PHONY: test-unit
+test-unit:
```

Comment:
Can you update the workflows to use makefile targets? If we don't test them on pull request someone might break them at some point.

Reply:
Good idea - done.  I needed to backport one more make recipe for `test-e2e` to handle this nicely, as for 386 architecture we don't publish releases thus don't run `make test-e2e-release` only `make test-e2e`.

Reply:
Test failure looks to be actions performance issue, hitting 15m timeout which we have often been close to historically.  Will retry later today.